### DISCUSSION TITLE
Service anymailfinder.com indexes your domain. SimpleLogin team need to contact them to remove SL domains from search

### DISCUSSION THREAD

**Discussion by CatBobN** [simple-login/app#1858] [Q&A]
Hello!
Service anymailfinder.com indexes your domain (screenshot attached), when the domain protonmail.com is excluded from the search because of that is a personal email creating service (screenshot attached).

Please contact the anymailfinder.com team with a request to exclude your domains from the search as well. They exclude all domains that are using to create personal addresses (but they need an official request).

[Screenshot 1](https://github.com/ProtonMail/WebClients/assets/143723119/01831119-1e19-4713-b67e-0bce8c5c0a74)
[Screenshot 2](https://github.com/ProtonMail/WebClients/assets/143723119/43c75824-7c52-42d2-90a8-d260541cb73d)
[Screenshot 3](https://github.com/ProtonMail/WebClients/assets/143723119/aa9d8a65-8cc4-49b4-bf18-0222a5cfce4f)

Also they are indexing ALL SimpleLogin and ProtonPass domains. Here is the list of indexed domains:

```
protonmail.ch
proton.me
pm.me
simplelogin.com
silomails.com
slmails.com
simplelogin.fr
aleeas.com
slmail.me
8shield.net
dralias.com
passinbox.com
passfwd.com
passmail.com
passmail.net
```

Please contact them to resolve this as soon as possible!

@acasajus and @nguyenkims take a look

**Comment by nguyenkims** [comment]
@CatBobN thanks, this seems to be a duplicated of another post?

**Comment by Tapi894** [comment]
@nguyenkims you can use this form to submit requests: https://app.anymailfinder.com/opt_out

**Reply by nguyenkims** [reply]
Thanks, we'll do it.

**Comment by acasajus** [comment]
It's done :+1:
